BALSAMIC/config/balsamic_env.yaml

```
@@ -28,4 +28,4 @@ varcall_cnvkit:
 vcf_merge:
   - vcfmerge
 varcall_delly:
-  - delly
+  - delly
```

Add ascatngs here as well.
```suggestion
  - delly
  ascatngs:
     - ascat
```